shell/docker-push.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Ensure we fail the job if any steps fail
+set -e -o pipefail
+
+# Push our image to wherever it's defined to go by the name
+docker push "$DOCKER_IMAGE"
```

Comment:
No reason to, this is being done on a single shot instance. It will automatically get cleaned up as soon as the job is done.